ON-CALL.md

```diff
@@ -0,0 +1,60 @@
+# Istio On-call Playbook
+
+## Who
+Every week, one Istio developer is on-call and is responsible for maintaining Istio build and for helping out users and other developers. The on-call person should prioritize on-call duties on top of the daily work.
+
+## Schedule
+The on-call schedule can be found [here](https://docs.google.com/spreadsheets/d/1FaHwPpad3F3hva2suJweNeTnocjKtLnbgLkyMRPzgUY/edit#gid=1475801904).
+
+On-call duty starts on Tuesday at noon PST, ends the following week on Tuesday at noon PST, and is performed during regular working hours for your time zone.
```

Comment:
so, for non-PST timezoned folks, this implies that their oncall cycle might start at the beginning of their regular working hours some time thereafter, correct?  and their cycles would end before the handoff? 

For the oncall members in Israel, noon PST is like 10PM and 10AM is midnight PST. I'm not sure how much this matters, but is worth mentioning, especially as it regards oncall expectations.

Reply:
Right, it is kind of implied by the business hours I mention previously.